pages/Berries/jaboca.md

```diff
@@ -0,0 +1,49 @@
+# Jaboca
+
+Jaboca is a type of [Berry](https://wiki.pokeclicker.com/) that can be planted in the [Farm](https://wiki.pokeclicker.com/). It can be obtained by having at least one empty plot and at least 600 points of bitter on the field. The [Durin](https://wiki.pokeclicker.com/) must be unlocked to unlock this berry. Various [Farming Setups](https://wiki.pokeclicker.com/) using Jaboca can be found [here](https://wiki.pokeclicker.com/).
+
+ ## Table of Contents
+ 1. [Growth Tables](#growth_tables)
```

Comment:
Unfortunately we can't use `#` links, as we are using `#!` for page links.
Will look into a way we can do this though.

Reply:
Never mind, resolved this..

You can link using `#id-here` and to create an id to scroll to just use `{#id-here}`

For example:
```md
# My title {#title}

[scroll to title](#title)
```